tests/framework/clients/rancher/provisioning/cluster.go

```diff
@@ -2,22 +2,147 @@ package provisioning
 
 import (
 	"context"
+	"time"
 
-	apisV1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
+	v1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
+	scheme "github.com/rancher/rancher/pkg/generated/clientset/versioned/scheme"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 )
 
-// Create is ClusterInterface's Create function, that is being overwritten to register its delete function to the session.Session
-// that is being reference.
-func (c *Cluster) Create(ctx context.Context, cluster *apisV1.Cluster, opts metav1.CreateOptions) (*apisV1.Cluster, error) {
+const (
```

Comment:
why do we need this constant here?

Reply:
Because this is the Steve api endpoint we need to hit